cclib/parser/data.py

```diff
@@ -35,7 +35,7 @@ class ccData(object):
         charge -- net charge of the system (integer)
         coreelectrons -- number of core electrons in atom pseudopotentials (array[1])
         enthalpy -- sum of electronic and thermal enthalpies (float, hartree/particle)
-        entropy -- entropy (float, hartree/particle)
+        entropy -- entropy (float, hartree/particle*kelvin)

```

Comment:
1. Is it customary to add the "particle" bit for entropy?
2. Shouldn't his be hartree/(particle*Kelvin) - currently looks a little like K is in the numerator if read programmatically.

Reply:
>     1. Is it customary to add the "particle" bit for entropy?

I used to keep consistency with [the docs](https://cclib.github.io/data.html). I think it is fine if we remove.

>     2. Shouldn't his be hartree/(particle*Kelvin) - currently looks a little like K is in the numerator if read programmatically.

You're right. I'll fix that.

Reply:
I don't mind. Actually I would defer to your judgement whether the particle bit should stay, just would press on trying to keep being consistent throughout the docs and any comments (it's always a struggle).

Reply:
OK. So I would say to just add the parenthesis and leave the "particle" as it is.

Reply:
Sounds good.